src/client/src/serviceWorker.js

```diff
@@ -55,7 +54,7 @@ export function register(config) {
 function registerValidSW(swUrl, config) {
   navigator.serviceWorker
     .register(swUrl)
-    .then(registration => {
+    .then((registration) => {
```

Comment:
Minor: Is there a reason you changed these parameters to be wrapped in brackets @pmilla1606, or was that done by prettier? I can't remember what our convention is, but I prefer single parameters not to have brackets!

Reply:
yup - that was prettier!